src/test/scala/com.snowplowanalytics.forex/ForexAtSpec.scala

```diff
@@ -28,15 +32,15 @@ class ForexAtSpec extends Specification {
   /**
    * GBP->CAD with USD as baseCurrency
    */
-  val tradeDate = new DateTime(2011, 3, 13, 11, 39, 27, 567, DateTimeZone.forID("America/New_York"))
+  val tradeDate = ZonedDateTime.of(2011, 3, 13, 11, 39, 27, 567, ZoneId.of("America/New_York"))
 
   val gbpToCadWithBaseUsd = fx.rate("GBP").to("CAD").at(tradeDate)
 
-  val cadMoney = gbpToCadWithBaseUsd.right.get
+  val cadMoney = gbpToCadWithBaseUsd
 
-  "GBP to CAD with USD as base currency returning latest eod rate [%s]".format(cadMoney) should {
+  "GBP to CAD with USD as base currency returning latest eod rate" should {
     "be > 0" in {
-      cadMoney.isPositive
+      gbpToCadWithBaseUsd.map(_ must beRight((m: Money) => m.isPositive)).unsafeRunSync()
```

Comment:
this is just stylistic but aren't we better of with an `_.unsafeRunSync() must beRight()`?

Reply:
this might be more readable